softwarereview_policies.Rmd

```diff
@@ -31,6 +31,19 @@ Last but not least, you'll find the code of conduct of rOpenSci Software Peer Re
     is also under review at another venue, as this may result on conflicting
     requests for changes.
 
+### Conflict of interest for reviewers/editors {#coi}
+
+Following criteria define people who cannot serve as an editor or reviewer for a submission.
+
+* The authors with a major role are from the potential reviewer/editor's institution or institutional component (e.g., department)
+* Within in the past three years, the potential reviewer/editor has been a collaborator or has had any other professional relationship (e.g., served as a mentor) with any person on the package who has a major role
```

Comment:
I'd remove mentor here and add another bullet at the end:

- and lifetime for the family members, business partners, and thesis student/advisor or mentor